.github/workflows/container-image.yml

```diff
@@ -46,7 +46,17 @@ jobs:
       uses: ishworkh/docker-image-artifact-upload@v1
       with:
         image: "samba-server:latest"
+
         retention_days: 1
+  build-server-centos:
+    runs-on: ubuntu-latest
+    env:
+      BUILDAH_FORMAT: oci
```

Comment:
Is this line needed? isn't `oci` the default format?

Reply:
> Is this line needed? isn't `oci` the default format?

just copy and paste, but I was wondering the same, @synarete

Reply:
It's probably vestigial from earlier experiments.

Reply:
I commented it out now